packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -65,7 +65,6 @@ export const Components = {
         connectionSettings: 'Data source connection URL', // aria-label in grafana experimental
         manageAlerts: 'prometheus-alerts-manager', // id for switch component
         scrapeInterval: 'data-testid scrape interval',
-        queryTimeout: 'data-testid query timeout',
```

Comment:
We are a bit hesitant to remove things from the selectors package but can't find any other usages of this so should be good to go! 👍🏻